src/_layouts/base.njk

```diff
@@ -17,108 +17,94 @@ date: Last Modified
   {%- endif %}
 </head>
 <body>
-  <aside data-status="{{ status.type }}">
-    {{ status.text | md | safe }}
-  </aside>
-
-  <div class="controls">
-    <button data-toggle="mode">colors</button>
-  </div>
+  {% include "base/nav.njk" %}
 
   <header>
-    {% include "nav.njk" %}
-    {% if not (page.url == '/') %}
-      {% set latest = changes[0].time or draft or page.date %}
-      <time datetime="{{ latest.toISOString() }}">
-        Updated {{ latest.toLocaleString('en-US', {'dateStyle': 'medium'}) }}
+    <h1>{{ title | mdInline | safe }}</h1>
+
+    {% if tags %}
+      <time datetime="{{ page.date.toISOString() }}">
+        Updated {{ page.date.toLocaleString('en-US', {'dateStyle': 'medium'}) }}
```

Comment:
Do you really want to use `page.date` here? Because the `list.all` uses the `get.latest` filter which uses `page.data.changes[0].time` or `page.data.draft` etc, most of the items on the home page have meaningful dates, but then when you click through the date at the top of the index page is a much more recent file-last-modified date. For example, `CSS Scope & Encapsulation – updated Mar 20, 2023` on the home page, but then `Updated Sep 19, 2023` (for me locally) on the CSS Scope & Encapsulation page.